modules/cluster/src/main/java/io/fluo/cluster/util/LogbackUtil.java

```diff
@@ -34,42 +33,26 @@
 public class LogbackUtil {
   
   private static final Logger log = LoggerFactory.getLogger(LogbackUtil.class);
-  private static final String LOG_APPLICATION_PROP = FLUO_PREFIX + ".log.application";
-  private static final String LOG_DIR_PROP = FLUO_PREFIX + ".log.dir";
-  private static final String LOG_LOCAL_HOSTNAME_PROP = FLUO_PREFIX + ".log.local.hostname";
-  
-  public static void init(String application, String configDir, String logOutput) throws IOException {
-    init(application, configDir, logOutput, true);
-  }
 
-  public static void init(String application, String configDir, String logOutput, boolean debug) throws IOException {
-    
-    String logConfig;
-    
-    if (logOutput.equalsIgnoreCase("STDOUT")) {
-      // Use a specific log config, if it exists
-      logConfig = String.format("%s/logback-stdout-%s.xml", configDir, application);
-      if (!new File(logConfig).exists()) {
-        // otherwise, use the generic config
-        logConfig = String.format("%s/logback-stdout.xml", configDir);
-      }
-    } else {
-      
-      System.setProperty(LOG_APPLICATION_PROP, application);
-      System.setProperty(LOG_DIR_PROP, logOutput);
+  private static final String FLUO_LOG_APP = FLUO_PREFIX + ".log.app";
+  private static final String FLUO_LOG_DIR = FLUO_PREFIX + ".log.dir";
+  private static final String FLUO_LOG_HOST = FLUO_PREFIX + ".log.host";
 
-      String localhost = InetAddress.getLocalHost().getHostName();
-      System.setProperty(LOG_LOCAL_HOSTNAME_PROP, localhost);
- 
-      // Use a specific log config, if it exists
-      logConfig = String.format("%s/logback-file-%s.xml", configDir, application);
-      if (!new File(logConfig).exists()) {
-        // otherwise, use the generic config
-        logConfig = String.format("%s/logback-file.xml", configDir);
-      }
-    } 
-    
+  public static void init(String application, String configDir, String logDir) throws IOException {
+
+    String logConfig = String.format("%s/logback.xml", configDir);
     ClusterUtil.verifyConfigPathsExist(logConfig);
+
+    System.setProperty(FLUO_LOG_APP, application);
+    System.setProperty(FLUO_LOG_DIR, logDir);
+
+    String localHostname = InetAddress.getLocalHost().getHostName();
+    String instanceId = System.getenv("TWILL_INSTANCE_ID");
```

Comment:
Unfortunately, LogbackUtil is needed to reset the logging configuration to stop logback from using a Kafka appender.  When Twill makes sending to a Kafka broker in the application master configurable, we can probably just configure logging in the in logback.xml file and remove this class.